TombEngine/Game/effects/effects.cpp

```diff
@@ -439,7 +439,172 @@ void TriggerCyborgSpark(int x, int y, int z, short xv, short yv, short zv)
 
 void TriggerExplosionSparks(int x, int y, int z, int extraTrig, int dynamic, int uw, int roomNumber)
 {
-	TriggerExplosion(Vector3(x, y, z), 512, true, false, true, roomNumber);
+	int dx, dz, mirror, i, scalar;
+	unsigned char extras_table[4];
+	unsigned char r, g, b;
+
+	extras_table[0] = 0;
```

Comment:
instead of such initialization, you may do `unsigned char extras_table[4] = { 0, 4, 7, 10 };`

Reply:
fixed